Libraries/Utilities/BackAndroid.android.js

```diff
@@ -44,13 +44,16 @@ RCTDeviceEventEmitter.addListener(DEVICE_BACK_EVENT, function() {
  *
  * Example:
  *
- * ```js
+ * ```javascript
  * BackAndroid.addEventListener('hardwareBackPress', function() {
- * 	 if (!this.onMainScreen()) {
- * 	   this.goBack();
- * 	   return true;
- * 	 }
- * 	 return false;
+ *  // this.onMainScreen and this.goBack are just examples, you need 
+ *  // to implement them on your own or use your own implementation here
```

Comment:
It sounds a bit redundant. I'll change `you need to implement them on your own or use your own implementation here` to `you need to use your own implementation here`

I think we can even add a line about `navigator`.